packages/cards/src/TemplateCard.tsx

```diff
@@ -52,7 +63,7 @@ export const TemplateCard = ({
         zIndex="hide"
       />
 
-      <Box pos="absolute" top="1" right="1" zIndex="docked">
+      <Box pos="absolute" top="1" title="options" right="1" zIndex={theme.zIndices.docked + 1}>
```

Comment:
Why is the `title` hardcoded?
If we do that, which is definitely fine, we should just prepare ourselves directly to be multilingual.

Reply:
well I didnt saw a i18n or similar in the codebase, but yes we need locales!
in StateDot.tsx the state also declares `title` and is hardcoded
any suggestions?

Reply:
Yes, correct. I had only added the cases as placeholders. So I didn't really think about it there.
And no. At the moment, I don`t have any proposal. I will add `TODO` statements and open a Jira task for it. We can think about it at a later stage.